amq-online/README.md

```diff
@@ -36,9 +36,7 @@ To meet the description use cases outlined above a number of addresses need to b
 
 First, we need remove the standard authservice and to create a none authentication service.
 ```
-oc project openshift-enmasse
-oc delete authenticationservice standard-authservice
-oc create -f enmasse/authservice/none-authservice.yaml
+oc project redhat-rhmi-amq-online
```

Comment:
Actually, nothing seems to be done in "redhat-rhmi-amq-online" project, so this can be removed together with the description above.